hack/run-update-testing.sh

```diff
@@ -29,4 +28,4 @@ echo "Running Functional Tests: ${GINKGO_SUITS}"
 # --failFast: ginkgo will stop the suite right after the first spec failure
 # --flakeAttempts: rerun the test if it fails
 # -requireSuite: fail if tests are not executed because of missing suite
-GOFLAGS=-mod=vendor ginkgo $NO_COLOR --v -r --failFast --flakeAttempts=2 -requireSuite ${GINKGO_SUITS} -- -junitDir /tmp/artifacts
+GOFLAGS=-mod=vendor ginkgo $NO_COLOR --v -r --fail-fast --flake-attempts=2 -require-suite ${GINKGO_SUITS} --junit-report=report.xml
```

Comment:
Ginkgo V2 updates (https://onsi.github.io/ginkgo/MIGRATING_TO_V2) 
"Finally, the default timeout has been reduced from 24h down to 1h. Users with long-running tests may need to adjust the timeout in their CI scripts."

"In V2 the timeout now applies to the entire test suite run so that ginkgo -r -timeout=1h is now guaranteed to exit after (about) one hour."

@marioferh  Are we setting a default timeout somewhere else ?   if not then we should add the timeout explicitly 
(starting with 5h to give enough wiggle room)